src/store.ts

```diff
@@ -0,0 +1,22 @@
+import LruCache from "lru-cache";
```

Comment:
I'd call the file `cache` rather than `store`.

Reply:
Yeah I agree, will rename it

Reply:
50b15b4fd0bd5a30461342f726bf4b24810118f9